examples/tutorials/tutorial_2dcrm.py

```diff
@@ -0,0 +1,215 @@
+import flow360 as fl
+from flow360.component.geometry import Geometry
+from flow360.component.simulation import cloud
+
+from flow360.component.simulation.meshing_param.params import (
+    MeshingDefaults,
+    MeshingParams,
+)
+from flow360.component.simulation.meshing_param.volume_params import (
+    AutomatedFarfield,
+    UniformRefinement
+)
+from flow360.component.simulation.meshing_param.face_params import SurfaceRefinement
+from flow360.component.simulation.meshing_param.edge_params import (
+    SurfaceEdgeRefinement,
+    HeightBasedRefinement,
+    ProjectAnisoSpacing
+)
+from flow360.component.simulation.models.surface_models import (
+    Freestream,
+    Wall,
+    SlipWall
+)
+from flow360.component.simulation.models.volume_models import Fluid
+from flow360.component.simulation.models.solver_numerics import (
+    NavierStokesSolver,
+    LinearSolver,
+    SpalartAllmaras
+)
+from flow360.component.simulation.operating_condition.operating_condition import (
+    create_operating_condition_from_mach_reynolds
+)
+from flow360.component.simulation.primitives import (
+    ReferenceGeometry,
+    Cylinder
+)
+from flow360.component.simulation.simulation_params import SimulationParams
+from flow360.component.simulation.time_stepping.time_stepping import (
+    Steady,
+    RampCFL
+)
+from flow360.component.simulation.outputs.outputs import (
+    SurfaceOutput,
+    VolumeOutput
+)
+from flow360.component.simulation.unit_system import SI_unit_system, u
+from flow360.examples import Tutorial_2dcrm
+
+"""
+This is a tutorial case based on the "7.3. RANS CFD on 2D High-Lift System Configuration Using the Flow360 Python Client" available in the documentation linked below.
+
+https://docs.flexcompute.com/projects/flow360/en/latest/tutorials/Multielement_Configuration/Multielement_Configuration.html
+
+In this case we are looking at a 3-element airfoil, which is a cross-section of the NASA CRM-HL configuration.
+
+Settings defined in this file are derived from the documentation and aim to represent the old tutorial case using the new Flow 360 python client.
```

Comment:
let's not mention that this is meant to represent old tutorial. In few month time, this message will not make sense. You can mention it in description of PR

Reply:
fixed it

Reply:
I think this sentence is unchanged